Program8.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Task_8
+{
+       internal class Program
+       {
+           static void Main(string[] args)
+           {
+               Console.WriteLine("Введите значение x для функции y=f(x)");
+               var x = double.Parse(Console.ReadLine());
+
+               Console.WriteLine($"f({x}) = {MyFunction(x)}");
+               Console.ReadKey();
+           }
+
+           static double MyFunction(double x)
+           {
+               if ( Math.Abs (Math.Sin(x)) > x)
+                   return Math.Sin(x);
+               else if ( Math.Abs( Math.Sin(x)) < x)
+                   return -1*Math.Sin(x);
```

Comment:
Можно просто
```suggestion
                   return -Math.Sin(x);
```
Операция "-" (смена знака) имеется